jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -129,7 +129,7 @@ const ComponentRenderersMixin = superclass => class extends superclass {
         changeState={this._bound.changeState}
         displayTerminalButton={this._enableTerminal()}
         exportEnvironmentFile={
-          initUtils.exportEnvironmentFile.bind(initUtils, this.db)}
+          initUtils.exportEnvironmentFile.bind(initUtils, this.db, this.sendAnalytics)}
```

Comment:
Is sendAnalytics already bound?

Reply:
Everything it needs is either a global or passed in on call so it doesn't need to be bound.